src/login/logind-action.c

```diff
@@ -299,12 +300,41 @@ static int handle_action_sleep_execute(
         return handle_action_execute(m, handle, ignore_inhibited, is_edge);
 }
 
+static int manager_handle_action_secure_attention_key(
+                Manager *m,
+                bool is_edge,
+                char *seat) {
+        int r;
+        Seat *o;
+        char *p;
+
```

Comment:
we generally validate function params with assert()s for non-Null-ness.

I.e. add `assert(m);` here and `assert(seat);`, since you expect both of them to be supplied.